include/lsst/jointcal/Associations.h

```diff
@@ -138,6 +138,17 @@ class Associations {
                 [](std::shared_ptr<CcdImage> const &item) { return item->getCatalogForFit().size() > 0; });
     }
 
+    /**
+     * @brief      Return the number of fittedStars that have an associated refStar.
+     */
+    size_t nFittedStarsWithAssociatedRefStar() {
```

Comment:
This should be a const method  since it does not alter the object (add `const` before `{`)

Also, please consider moving the implementation to `Associations.cc`.